### DISCUSSION TITLE
Type error: Could not find a declaration file for module '@mdx-js/runtime'

### DISCUSSION THREAD

**Discussion by sajrashid** [mdx-js/mdx#1778] [Q&A] [answered]
Hello,

I'm running next, works in dev,  build throws the error :(

tried, 

 `npm i --save-dev @types/mdx-js__rs__runtime` 

**returns 404** 

I was getting similar with  `mdx-js__react`

running `npm` installed it OK

 package.json entries

 "dependencies":
"@mdx-js/react": "^1.6.22",
"@mdx-js/runtime": "^1.6.22",

"devDependencies":
"@types/mdx-js__react": "^1.5.4",

Idea's greatly appreciated ?

Thanks.

**Comment by wooorm** [comment, marked as answer]
We have types in V2. See https://v2.mdxjs.com/blog/v2/ and https://v2.mdxjs.com/docs/getting-started/#types.
